### PR TITLE
Update 2.4.1 release notes

### PR DESCRIPTION
**Pull request by sravyap135**
**Description**:  Update 2.4.1 release notes, Add resolves keyword to automatically close GITHUB issues.

### CODE REVIEW

**Review by vklohiya — APPROVED** [review]
Looks Fine.

File: docs/RELEASE-NOTES.rst [hunk 1]
```diff
@@ -2,22 +2,34 @@ Release Notes for Container Ingress Services for Kubernetes & OpenShift
...
 * :issues: `1808` K8S BIG-IP Controller upload old certificate to BIG-IP.
-* Fix Deleting Stale Specs from IPAM CR when CIS restarts
-* Fix Populate allocated ip for virtual-server in the VSAddress colomn while using IPAM.
-* Fix Processing endpoint with node names when CIS in Cluster mode
-
-Limitations
-```````````
+* Stale IPAM CR configuration is deleted when CIS restarts.
+* IPAM allocated IP address now populates for VirtualServer under VSAddress column.
+* CIS supports endpoints created with no nodeNames in Cluster mode for Headless Service.
```
Comment (trinaths) [inline docs/RELEASE-NOTES.rst:19]: CIS supports endpoints created without nodeNames in Cluster mode for Headless Service.

**Review by trinaths — CHANGES_REQUESTED** [review]

File: docs/RELEASE-NOTES.rst (hunk 1 above)
Comment (sravyap135) [reply docs/RELEASE-NOTES.rst]: Done

**Review by trinaths — APPROVED** [review]
looks good to me

**Pull request merged**